### DISCUSSION TITLE
The docs for the manifest don't match the schema for ExpectedReturnCodes

### DISCUSSION THREAD

**Discussion by corbob** [microsoft/winget-pkgs#115765] [General]
If you look at this part of the docs for the manifest: https://github.com/microsoft/winget-pkgs/blob/c4542f57796686a9072d1f09d5e5ef8b515bb2fd/doc/manifest/schema/1.4.0/installer.md?plain=1#L70, it says that the element name is `ExtpectedReturnCode`, but if you use this, then `winget validate -m .` will fail, as the Schema says that it is `InstallerReturnCode`: https://github.com/microsoft/winget-cli/blob/abd0f94866e51a0e64c5aa363248dd7da321aa15/schemas/JSON/manifests/v1.4.0/manifest.installer.1.4.0.json#L218.

(Note: this appears to be a discrepancy across all of the schema versions, so would likely be good to verify and update them all)

**Comment by denelon** [comment]
I created:
* #116421
